lib/db/schema/patch-022-023.sql

```diff
@@ -0,0 +1,133 @@
+-- This migration:
+--
+-- * Changes the type of the devices.callbackPublicKey column
+-- * Add a new devices.callbackAuthKey column
+-- * Updates the accountDevices stored procedure with the new column
+-- * Alters three stored procedures, createDevice, updateDevice and
+--   sessionWithDevice to work with the new columns
+
+ALTER TABLE devices
+MODIFY COLUMN callbackPublicKey VARCHAR(88),
```

Comment:
Will this apply cleanly to rows that have existing data in this column?  (Whether we have any is a separate question).

In general migrations that do a `MODIFY COLUMN` are hard to deploy, because you will still have old versions of the code running that expect to use the old column definition.  We'll likely have to do this as a multi-step procedure:

1.  Add a new column for the new-format data, and if possible do some query to populate it based on existing data in the old column.
2.  Deploy the migration and the code that starts using the new column.
3.  Once that's running successfully in production, do another migration that drops the original column.